extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -1244,10 +1245,13 @@ class LifecycleTask extends BackbeatTask {
                      daysSinceInitiated >= rules.Expiration.Days) ||
                     (rules.Expiration.Date !== undefined &&
                      rules.Expiration.Date < Date.now()) ||
+                    // TODO: BB-376 why do we check that eodm is "not explicitly set to false" instead
```

Comment:
@jonathan-gramain do you know why? Thanks.
https://scality.atlassian.net/browse/BB-376